QUEST_LV_0100.tsv

```diff
@@ -4198,176 +4198,176 @@ QUEST_LV_0100_20150317_004197	$Destroy the West Barrier in Nuotori Cliff
-QUEST_LV_0100_20150317_004344	Use Mining Compass
-QUEST_LV_0100_20150317_004345	Use mining compass to check what you need to fix the secondary purifier.
-QUEST_LV_0100_20150317_004346	Check the spell device
-QUEST_LV_0100_20150317_004347	 The mine compass is pointing to the spell device in 4th area. Check what the problem is. 
-QUEST_LV_0100_20150317_004348	Use mining compass to check what you need to fix the secondary purifier.
-QUEST_LV_0100_20150317_004349	Search for tools to remove rust
-QUEST_LV_0100_20150317_004350	Follow the compass to find a way to remove the rust.
-QUEST_LV_0100_20150317_004351	Restore the spell device in district 4
-QUEST_LV_0100_20150317_004352	Seems like you can fix the spell device in Area 4 with lubricant.
-QUEST_LV_0100_20150317_004353	Activate secondary purifier in 2nd Gallery
-QUEST_LV_0100_20150317_004354	Seem like spell supply is working well. Activate the secondary purifier.
-QUEST_LV_0100_20150317_004355	Check the main purifier
-QUEST_LV_0100_20150317_004356	Main purifier is not working. Go and check on it.
-QUEST_LV_0100_20150317_004357	Use Mine Compass to locate the lost parts
+QUEST_LV_0100_20150317_004309	$Repair the Purifiers on the 2nd Floor
+QUEST_LV_0100_20150317_004310	$Repair all the Purifiers on the 2nd Floor
+QUEST_LV_0100_20150317_004311	$Inspect the Central Purifier
+QUEST_LV_0100_20150317_004312	$The Mine Compass is pointing towards the Central Purifier on the 1st floor. Inspect the purifier. 
+QUEST_LV_0100_20150317_004313	$Use the Mine Compass
+QUEST_LV_0100_20150317_004314	$Use the Mine Compass to find the Spare Purifier located somewhere on the 1st floor.
+QUEST_LV_0100_20150317_004315	$Search District 4 for the Spare Purifier
+QUEST_LV_0100_20150317_004316	$The Mine Compass is pointing towards District 4. Follow its lead and find the Spare Purifier that can be used to repair the Purifier.
+QUEST_LV_0100_20150317_004317	$Get parts from the Spare Purifier in District 4.
+QUEST_LV_0100_20150317_004318	$Get the replacement parts from the Spare Purifier.
+QUEST_LV_0100_20150317_004319	$Obtained spare parts for the Central Purifier. Replace the parts of the Purifier.
+QUEST_LV_0100_20150317_004320	$Defeat Cyclops in the Crystal Mine
+QUEST_LV_0100_20150317_004321	$Cyclops is rampaging in the Crystal Mine. Defeat Cyclops. 
+QUEST_LV_0100_20150317_004322	$Defeat Cyclops
+QUEST_LV_0100_20150317_004323	$Inspect the Passage Purifier in 1st Floor
+QUEST_LV_0100_20150317_004324	$There is a problem with the purifier on the path to Crystal Mine 2F. Go and inspect it.
+QUEST_LV_0100_20150317_004325	$Use the Mine Compass to find the parts needed to repair the purifier.
+QUEST_LV_0100_20150317_004326	$Search District 6 for Purifier Parts
+QUEST_LV_0100_20150317_004327	$The Mine Compass is pointing towards District 6. Follow it and find parts to repair the purifier.
+QUEST_LV_0100_20150317_004328	 Retrieve %s from Specter Monarch
```

Comment:
@Sourpusss has a PR that works on this line. I just merely matched what was on his PR.